x-pack/legacy/plugins/siem/public/lib/connectors/servicenow.tsx

```diff
@@ -98,6 +98,21 @@ const ServiceNowConnectorFields: React.FunctionComponent<ActionConnectorFieldsPr
   const isUsernameInvalid: boolean = errors.username.length > 0 && username != null;
   const isPasswordInvalid: boolean = errors.password.length > 0 && password != null;
 
+  /**
```

Comment:
That's really nice explanation, future developer going to love it

Reply:
Thank you!